ArduPlane/Parameters.h

```diff
@@ -266,8 +266,8 @@ class Parameters {
         k_param_throttle_suppress_manual,
         k_param_throttle_passthru_stabilize,
         k_param_rc_12_old,
-        k_param_fs_batt_voltage, // unused - moved to AP_BattMonitor
```

Comment:
You can't grab the old indexes as that breaks param conversion for people on older boards.